Library/Homebrew/diagnostic.rb

```diff
@@ -953,67 +702,6 @@ def check_for_bad_python_symlink
         EOS
       end
 
-      def check_for_non_prefixed_coreutils
-        coreutils = Formula["coreutils"]
-        return unless coreutils.any_version_installed?
-
-        gnubin = %W[#{coreutils.opt_libexec}/gnubin #{coreutils.libexec}/gnubin]
-        return if (paths & gnubin).empty?
-
-        <<~EOS
-          Putting non-prefixed coreutils in your path can cause gmp builds to fail.
-        EOS
-      rescue FormulaUnavailableError
-        nil
-      end
-
-      def check_for_non_prefixed_findutils
-        findutils = Formula["findutils"]
-        return unless findutils.any_version_installed?
-
-        gnubin = %W[#{findutils.opt_libexec}/gnubin #{findutils.libexec}/gnubin]
-        default_names = Tab.for_name("findutils").with? "default-names"
-        return if !default_names && (paths & gnubin).empty?
-
-        <<~EOS
-          Putting non-prefixed findutils in your path can cause python builds to fail.
-        EOS
-      rescue FormulaUnavailableError
-        nil
-      end
-
-      def check_for_pydistutils_cfg_in_home
-        return unless File.exist? "#{ENV["HOME"]}/.pydistutils.cfg"
-
-        <<~EOS
-          A .pydistutils.cfg file was found in $HOME, which may cause Python
-          builds to fail. See:
-            #{Formatter.url("https://bugs.python.org/issue6138")}
-            #{Formatter.url("https://bugs.python.org/issue4655")}
-        EOS
-      end
-
-      def check_for_unlinked_but_not_keg_only
```

Comment:
This still isn't a supported or persistent configuration.

Reply:
@ilovezfs Agreed but is it a cause of issues/bugs? If so: I'll add it back.

Reply:
Yes. This is why we require formulae to be linked during the test block.